assets/js/components/PageHeader.component.js

```diff
@@ -38,6 +38,16 @@ export default class PageHeader {
     this.navToggle.addEventListener('click', (event) => {
       event.preventDefault();
 
+      let ariaExpanded = this.navToggle.getAttribute('aria-expanded');
```

Comment:
to možeš skratiti na: 
   let ariaExpanded = this.navToggle.getAttribute('aria-expanded') === 'true';

   this.navToggle.setAttribute('aria-expanded', !ariaExpanded);